### PR TITLE
BGA-435 Add Tron WBTC and WETH configuration to Statics

### PR DESCRIPTION
**Pull request by silalvado**
Add configurations for WBTC and WETH tokens deployed on Tron network.

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -499,6 +501,9 @@ export const coins = CoinMap.fromCoins([
     '',
     'WXT'
   ),
+  // TODO Set correct contract addresses for WBTC and WETH on Tron main network
+  tronToken('trx:wbtc', 'Tron Wrapped Bitcoin', 8, 'TPamvhorQcFpvtMtYb6qDDJNMsUM1doPty', UnderlyingAsset.WBTC),
```
Comment (twtaylorbitgo) [inline modules/statics/src/coins.ts:505]: I'm not sure these UnderlyingASsets are right, I'll have to get back to this

**Review by twtaylorbitgo — CHANGES_REQUESTED** [review]
Overall LGTM - I just have to check out how underlyingasset is used in our arch.

File: modules/statics/src/coins.ts (hunk 1 above)
Comment (silalvado) [reply modules/statics/src/coins.ts]: Ok. I set two example addresses for WBTC and WETH in mainnet.

**Comment by tylerlevine** [comment]
Hi @silalvado, is this PR still active or can it be closed?

**Comment by FabianSimtlix** [comment]
This was deprecated. Closing the PR

**Pull request closed without merging**